.github/workflows/ci.yaml

```diff
@@ -356,7 +356,7 @@ jobs:
       - name: "Start kind cluster"
         run: |
           kind version
-          kind create cluster --image kindest/node:v1.21.1@sha256:69860bda5563ac81e3c0057d654b5253219618a22ec3a346306239bba8cfa1a6 --wait 3m
+          kind create cluster --image kindest/node:v1.22.0@sha256:b8bda84bb3a190e6e028b1760d277454a72267a5454b57db34437c34a588d047
```

Comment:
Was removing the wait-flag intentional?

Reply:
Oh shit no 🙃

Added it back, and this time before the long hash so that I don't overlook it again next time 😀